tests/integration/RequestBuilder/EventsRequestBuilderTest.php

```diff
@@ -6,13 +6,37 @@
 use Lmc\Matej\IntegrationTests\IntegrationTestCase;
 use Lmc\Matej\Model\Command\Interaction;
 use Lmc\Matej\Model\Command\ItemProperty;
+use Lmc\Matej\Model\Command\ItemPropertySetup;
 use Lmc\Matej\Model\Command\UserMerge;
+use Lmc\Matej\RequestBuilder\ItemPropertiesSetupRequestBuilder;
 
 /**
  * @covers \Lmc\Matej\RequestBuilder\EventsRequestBuilder
  */
 class EventsRequestBuilderTest extends IntegrationTestCase
 {
+    private function buildAndSendPropertySetupRequest(ItemPropertiesSetupRequestBuilder $builder): void
```

Comment:
Commit 95c55be86ff6a93469e0251226ea8393f2486bd0 seems to be unrelated with this PR and should be separate PR. Or am I wrong?

Reply:
ahaha, that's because fresh instances don't have these item properties set up.
I can cherry-pick it into separate PR if you want. I don't really feel it's a problem though.